### ISSUE TITLE
Uncaught TypeError: Cannot read property 'maindir' of undefined

### ISSUE THREAD

**Issue by yspreen** [mnater/Hyphenopoly#24]
Please update the example in the readme with the example from example1.html.

The current example throws:  
`Uncaught TypeError: Cannot read property 'maindir' of undefined` In `Hyphenopoly_Loader.js:359`

Because `paths` is undefined in that example.

---

For everyone with the same issue, use this:

```
        var Hyphenopoly = {
            require: {
                "la": "honorificabilitudinitas",
                "de": "Silbentrennungsalgorithmus",
                "en-us": "Supercalifragilisticexpialidocious"
            },
            paths: {
                patterndir: "./patterns/",
                maindir: "./"
            },
            setup: {
                classnames: {
                    "container": {
                    }
                }
            }
        };
```

**Comment by mnater** [comment]
Thanks!

The example is correct: the paths should resolve to "./" and "./patterns" by default.
Fixed the code, left README as it is.

**Comment by lukehsiao** [comment]
The readme should be fixed to prevent people from running into this error.
> The Hyphenopoly object must have exactly one property called require which itself is an object containing at least one nameValuePair where the name is a language code string (Some patterns are region-specific. See the patterns directory for supported languages.

Right now the user must also define the paths and setup.

To match the behavior described in the README:
> Hyphenopoly by default hyphenates elements (and their children) with the classname .hyphenate. Don't forget to enable CSS-hyphenation for the classes eventually handled by Hyphenopoly.

```js
<script>
var Hyphenopoly = {
    require: {
        "en-us": "Supercalifragilisticexpialidocious"
    },
    paths: {
      patterndir: "./Hyphenopoly/patterns/",
      maindir: "./Hyphenopoly/"
    },
    setup: {
        classnames: {
            "hyphenate": {
            }
        }
    },
};
</script>
```

By default the current run() is not called:
https://github.com/mnater/Hyphenopoly/blob/aeefe6e3a59e8356abc99ca490acabf6c3374d7b/Hyphenopoly_Loader.js#L502-L523

That said, it appears to work well!

**Comment by mnater** [comment]
`run()` is a [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE)

I didn‘t understand what should be changed in the readme…

**Comment by lukehsiao** [comment]
run() is not called automatically for me in Chrome 67.0.3396.62, which meant that I needed to specify both the paths and the setup. If run is called automatically, then the README is probably accurate, but because it wasn't then there was a mismatch.

**Comment by torfsen** [comment]
@lukehsiao looks like this was fixed in aeefe6e3a59e8356abc99ca490acabf6c3374d7b.
